skietbaan_fe/src/components/Login.js

```diff
@@ -44,18 +57,18 @@ class App extends Component {
       ...stateUpdate ,
       validForm: isValid
     });
-  };
+  }
 
   Login() {
+    this.Validate();
     if (this.state.validForm) {
       let sha1 = require('sha1');  
       let hash = sha1(this.state.passwordValue);
       let RequestObject = {
         "Username": this.state.usernameValue,
-        "Email": this.state.emailValue,
         "Password": hash,
       }
-      fetch("http://localhost:63474/api/User", {
+      fetch("http://localhost:63474/api/features/login", {
```

Comment:
URLs should be in const file and should not be committed as localhost

Reply:
Ok fixed